selenium/sample-build.js

```diff
@@ -15,11 +15,18 @@
  * along with this program.  If not, see <https://www.gnu.org/licenses/>.
  */
 
+const fs = require("fs");
 const webdriver = require("selenium-webdriver");
+const chrome = require("selenium-webdriver/chrome");
```

Comment:
Codacy has a fix for the issue: [Do not use "require".](https://app.codacy.com/gh/pixiebrix/pixiebrix-extension/pullRequest?prid=7734019)
```suggestion
import chrome from "selenium-webdriver/chrome";
```